### PR TITLE
Add toolbox support to nuget package

### PR DESCRIPTION
**Pull request by jassmith**

### CODE REVIEW

File: toolbox/Xamarin.Forms.toolbox.xml [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+			</Snippet>
+		</Item>
+		<Item Class="Xamarin.Forms.ActivityIndicator" Name="ActivityIndicator" Image="view-ActivityIndicator-16.png">
+			<Snippet>
+				<ActivityIndicator />
+			</Snippet>
+		</Item>
+		<Item Class="Xamarin.Forms.Button" Name="Button" Image="view-Button-16.png">
+			<Snippet>
+				<Button Text="" />
```
Comment (StephaneDelcroix) [inline toolbox/Xamarin.Forms.toolbox.xml]: I'd do
```<Button Text="OK" />```
or better
```<Button Text="{Binding ButtonText}" />```

File: toolbox/Xamarin.Forms.toolbox.xml [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+			<Snippet>
+				<Image Source="" />
+			</Snippet>
+		</Item>
+		<Item Class="Xamarin.Forms.ListView" Name="ListView" Image="view-ListView-16.png">
+			<Snippet>
+				<ListView ItemsSource="">
+					<ListView.ItemTemplate>
+						<DataTemplate>
+						</DataTemplate>
```
Comment (StephaneDelcroix) [inline toolbox/Xamarin.Forms.toolbox.xml]: why not going one step further and adding the `ViewCell` ?

File: toolbox/Xamarin.Forms.toolbox.xml [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+			<Snippet>
+				<Entry Placeholder="" />
+			</Snippet>
+		</Item>
+		<Item Class="Xamarin.Forms.Image" Name="Image" Image="view-Image-16.png">
+			<Snippet>
+				<Image Source="" />
+			</Snippet>
+		</Item>
+		<Item Class="Xamarin.Forms.ListView" Name="ListView" Image="view-ListView-16.png">
```
Comment (StephaneDelcroix) [inline toolbox/Xamarin.Forms.toolbox.xml]: uh, where's `Label` ?

File: toolbox/Xamarin.Forms.toolbox.xml [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+			<Snippet>
+				<ListView ItemsSource="">
+					<ListView.ItemTemplate>
+						<DataTemplate>
+						</DataTemplate>
+					</ListView.ItemTemplate>
+				</ListView>
+			</Snippet>
+		</Item>
+		<Item Class="Xamarin.Forms.ProgressBar" Name="ProgressBar" Image="view-ProgressBar-16.png">
```
Comment (StephaneDelcroix) [inline toolbox/Xamarin.Forms.toolbox.xml]: and `Picker` ?

File: toolbox/Xamarin.Forms.toolbox.xml [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+			<Snippet>
+				<Stepper Minimum="0" Maximum="10" Increment="1" />
+			</Snippet>
+		</Item>
+		<Item Class="Xamarin.Forms.Switch" Name="Switch" Image="view-Switch-16.png">
+			<Snippet>
+				<Switch IsToggled="false" />
+			</Snippet>
+		</Item>
+		<Item Class="Xamarin.Forms.TableView" Name="TableView" Image="view-TableView-16.png">
```
Comment (StephaneDelcroix) [inline toolbox/Xamarin.Forms.toolbox.xml]: I wouldn't provide designer shortcuts for controls we don't really want to keep around

File: toolbox/Xamarin.Forms.toolbox.xml [hunk 6]
```diff
@@ -0,0 +1,167 @@
...
+			<Snippet>
+				<ScrollView></ScrollView>
+			</Snippet>
+		</Item>
+		<Item Class="Xamarin.Forms.AbsoluteLayout" Name="AbsoluteLayout" Image="layout-AbsoluteLayout-16.png">
+			<Snippet>
+				<AbsoluteLayout></AbsoluteLayout>
+			</Snippet>
+		</Item>
+		<Item Class="Xamarin.Forms.Grid" Name="Grid" Image="layout-Grid-16.png">
```
Comment (StephaneDelcroix) [inline toolbox/Xamarin.Forms.toolbox.xml]: you forgot the `F***Layout`

Comment (jassmith) [reply toolbox/Xamarin.Forms.toolbox.xml]: Not in 15-5 so yeah we dont include it here

File: toolbox/Xamarin.Forms.toolbox.xml (hunk 5 above)
Comment (jassmith) [reply toolbox/Xamarin.Forms.toolbox.xml]: its on the list provided by PM, they want it

File: toolbox/Xamarin.Forms.toolbox.xml (hunk 4 above)
Comment (jassmith) [reply toolbox/Xamarin.Forms.toolbox.xml]: fixed

File: toolbox/Xamarin.Forms.toolbox.xml (hunk 3 above)
Comment (jassmith) [reply toolbox/Xamarin.Forms.toolbox.xml]: fixed

File: toolbox/Xamarin.Forms.toolbox.xml (hunk 2 above)
Comment (jassmith) [reply toolbox/Xamarin.Forms.toolbox.xml]: Snippets provided by PM

File: toolbox/Xamarin.Forms.toolbox.xml (hunk 1 above)
Comment (jassmith) [reply toolbox/Xamarin.Forms.toolbox.xml]: Snippets provided by PM

**Pull request merged**